easybuild/easyconfigs/e/EvtGen/EvtGen-01.07.00-gompi-2022a-HepMC-2.06.11.eb

```
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'EvtGen'
+version = '01.07.00'
+local_hepmc = "HepMC"
+local_hepmcver = "2.06.11"
+versionsuffix = '-%s-%s' % (local_hepmc, local_hepmcver)
+
+homepage = 'http://evtgen.hepforge.org/'
+description = """
+EvtGen is a Monte Carlo event generator that simulates the decays of heavy flavour particles, primarily B and D mesons.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['https://evtgen.hepforge.org/downloads?f=']
+sources = [SOURCE_TAR_GZ]
+checksums = ['7ed109ac778a6fba229337d5dbe00e2d05bc5a25776969574f9575a863a4e22d']
+
+patches = ['EvtGen-01.07.00_patch-fix-herwig.patch']
+buildopts = '-j 1'  # parallel build fails
```

```suggestion
max_parallel = 1  # parallel build fails
```

The other way results in `make -j XXX -j 1`